src/Common.props

```diff
@@ -0,0 +1,55 @@
+<Project>
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <RuntimeIdentifiers>win-x64;win-x86;linux-x64;linux-arm;rhel.6-x64;osx-x64</RuntimeIdentifiers>
+    <TargetLatestRuntimePatch>true</TargetLatestRuntimePatch>
+    <AssetTargetFallback>portable-net45+win8</AssetTargetFallback>
+    <NoWarn>NU1701;NU1603;</NoWarn>
+    <OutputType>OVERRIDE_ME</OutputType>
+    <OSPlatform>OS_UNKNOWN</OSPlatform>
+    <OSArchitecture>ARCH_UNKNOWN</OSArchitecture>
+    <DebugConstant></DebugConstant>
+    <VssApiVersion>0.5.125-private</VssApiVersion>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <DebugType>portable</DebugType>
+    <DebugConstant>DEBUG</DebugConstant>
+
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::Windows)))' == 'true'">
+    <OSPlatform>OS_WINDOWS</OSPlatform>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::OSX)))' == 'true'">
+    <OSPlatform>OS_OSX</OSPlatform>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::Linux)))' == 'true'">
+    <OSPlatform>OS_LINUX</OSPlatform>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(OSPlatform)' == 'OS_WINDOWS' AND '$(PROCESSOR_ARCHITECTURE)' == 'AMD64'">
```

Comment:
where does PROCESSOR_ARCHITECTURE get set?

Reply:
i see, this is the built-in one, i don't think we should use this since you can build x86 agent on an x64 machine.
we better still use `$(PackageRuntime)` since that will help everything lines up

Reply:
This is just matching the existing bash logic here: https://github.com/microsoft/azure-pipelines-agent/blob/ca43afe47aae9d78606eeb8f602cdd2a5f5867af/src/dev.sh#L36